_includes/configuration/dockerized-hyperlinking-service.md

```diff
@@ -0,0 +1,134 @@
+{% elsif shbundledockerfiles == true %}
+### Deploy the TinyMCE hyperlinking server-side component using Docker (self-hosted enterprise bundle)
+{% endif %}
+The TinyMCE hyperlinking server-side component can be deployed using Docker or an container orchestration application such as Kubernetes. {{site.companyname}} provides a pre-configured package for creating a Hyperlinking Docker image.
+
+> **Warning**: Do not push this docker image to a publicly accessible container registry. Doing so will constitute a breach of the [{{site.companyname}} Self-Hosted Software License Agreement](https://about.tiny.cloud/legal/tiny-self-hosted-software-license-agreement-enterprise/).
+
+{% if shbundledockerfiles == false %}
+
+#### Prerequisites
+
+* The [Docker Engine](https://docs.docker.com/engine/docker-overview/) is installed and running.
+* The user has Administrative or Root user access.
+* The user is either:
+  * Using a Linux, Unix or macOS operating system.
+  * Using Windows and has access to unix command line tools using [Git for Windows](https://gitforwindows.org/), [Cygwin](https://www.cygwin.com/), or the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+{% endif %}
+
+#### Procedure
+
+The following procedure covers downloading, configuring, building and testing the {{site.productname}} Hyperlinking Docker image.
+{% if shbundledockerfiles == true %}
+> **Note**: This procedure assumes that the **{{ site.productname }} Enterprise Bundle** has been extracted as described in [Download and extract the TinyMCE Enterprise Bundle](#downloadandextractthetinymceenterprisebundle).
+
+4. Navigate into the `tinymce-services` folder.
+
+    ```sh
+    cd tinymce-enterprise/tinymce-services/
+    ```
+
+5. Extract the contents of `ephox-hyperlinking-docker-starter-kit.zip`, such as:
+
+    ```sh
+    unzip ephox-hyperlinking-docker-starter-kit.zip -d hyperlinking-service-dockerfile
+    ```
+
+6. Copy `ephox-hyperlinking.war` into the directory containing the extracted files, such as:
+
+    ```sh
+    cp ephox-hyperlinking.war hyperlinking-service-dockerfile/
+    ```
+
+4. Navigate into the `hyperlinking-service-dockerfile` folder.
+
+    ```sh
+    cd hyperlinking-service-dockerfile
+    ```
+{% elsif shbundledockerfiles == false %}
+1. Go to [{{ site.accountpage }}]({{ site.accountpageurl }}) > My Downloads
+and download either:
+* _Tiny Enhanced Media Embed_, or
+* _Tiny Link Checker_.
+2. Open a command line and navigate to the directory containing `ephox-hyperlinking_<version>.zip`. Windows Users should open a Bash command line as the Administrator User.
+3. Extract the contents of `ephox-hyperlinking_<version>.zip`, such as:
+
+    ```sh
+    unzip ephox-hyperlinking_<version>.zip -d tinymce-hyperlinking-service
+    ```
+    The structure of the extracted files will be:
+    ```sh
+    tinymce-hyperlinking-service/
+    ├── ephox-hyperlinking-docker-env.conf
```

Comment:
Placeholder reminder for either update the documentation to mention the `config` directory, or rebuild the docker starter kits before release.